backend/core/src/applications/applications.controller.ts

```diff
@@ -67,6 +79,44 @@ export class ApplicationsListQueryParams extends PaginationQueryParams {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   userId?: string
+
+  @Expose()
+  @ApiProperty({
+    enum: Object.keys(OrderByParamEnum),
+    example: "createdAt",
+    default: "createdAt",
+    required: false,
+  })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsIn(Object.values(OrderByParamEnum), { groups: [ValidationsGroupsEnum.default] })
+  @Transform(
+    (value: string | undefined) =>
+      value
+        ? OrderByParamEnum[value]
+          ? OrderByParamEnum[value]
+          : value
+        : OrderByParamEnum.createdAt,
+    {
+      toClassOnly: true,
+    }
+  )
+  orderBy?: OrderByParamEnum
+
+  @Expose()
+  @ApiProperty({
+    enum: OrderParamEnum,
+    example: "DESC",
+    default: "DESC",
+    required: false,
+  })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsIn(Object.keys(OrderParamEnum), { groups: [ValidationsGroupsEnum.default] })
+  @Transform((value: string | undefined) => (value ? value : OrderParamEnum.DESC), {
```

Comment:
Is `toClassOnly` required here since this is only ever used for input?

Reply:
Fixed